src/python/pants/backend/java/subsystems/java_infer.py

```diff
@@ -25,6 +25,8 @@ class JavaInferSubsystem(Subsystem):
         "--third-party-imports",
         default=True,
         help="Infer a target's third-party dependencies using Java import statements.",
+        removal_version="2.13.0.dev0",
+        removal_hint="Controlled by the `--imports` flag.",
```

Comment:
For posterity: this eager deprecation should be fine with our deprecation policy because this is an experimental backend. (normally the flag needs to still work)

Reply:
Yea, that was my thinking.